sdks/v4-sdk/src/utils/actions.ts

```diff
@@ -0,0 +1,75 @@
+import { defaultAbiCoder } from 'ethers/lib/utils'
+
```

Comment:
would be nice to keep router + posm actions using the same `encodeAction` code so whenever that gets merged maybe we re architect this so we have posm actions, router actions and then they share a "planner" file that does the encoding/building of actions

Reply:
yea i agree, ok we can wait on #87 for this one then. Talked to Ayo and they're ok with that

Reply:
ok actually going to remove this to unblock them